### PR TITLE
make cdap buildable with Java11

### PR DESCRIPTION
**Pull request by seanzhougoogle**

### CODE REVIEW

**Review by tivv — APPROVED** [review]
LGTM. Would love to see it works as I need JVM 11 build :)
May have problems with Unsafe in e.g. io.cdap.cdap.api.common.Bytes

**Comment by chtyim** [comment]
unit-test would fail in Java 11. See comment in https://github.com/cdapio/cdap/pull/12791#issuecomment-702977544

File: pom.xml [hunk 1]
```diff
@@ -159,7 +159,7 @@
     <quartz.version>2.2.0</quartz.version>
     <resteasy.version>3.0.8.Final</resteasy.version>
     <rs-api.version>2.0</rs-api.version>
-    <scala2.11.version>2.11.8</scala2.11.version>
+    <scala2.11.version>2.11.12</scala2.11.version>
```
Comment (chtyim) [inline pom.xml:162]: Just the scala version change is not enough. We also need to exclude the jdk.tools usage that brings in by the Hadoop dependency. See https://github.com/cdapio/cdap/pull/12791/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R597

**Review by chtyim — CHANGES_REQUESTED** [review]

**Comment by tivv** [comment]
Yes, I tried it and tons of test start to fail :(

File: pom.xml (hunk 1 above)
Comment (tivv) [reply pom.xml:162]: Yes. I would need it for sonalcloud. The build itself can be done with jdk8, but sonar analysis is done with jdk11 and this is where I get errors on jdk.tools

**Pull request closed without merging**